config/default.yml

```diff
@@ -912,16 +912,8 @@ Style/ParenthesesAroundCondition:
 
 Style/PercentLiteralDelimiters:
   PreferredDelimiters:
-    '%':  ()
```

Comment:
Although this is more or less clear you can add some comment here explaining the configuration of the cop.